Action: go to block position: Quaternion error

Another error in the behavior during our last game in the action: go to block position

## Expected behavior
Dont die

## Current behavior
```
[body_behavior-32] [ERROR] [1679050627.234098291] [dynamic_stack_decider]: Traceback (most recent call last):
[body_behavior-32]   File "/colcon_ws/install/dynamic_stack_decider/lib/python3.10/site-packages/dynamic_stack_decider/dsd.py", line 241, in update
[body_behavior-32]     result = current_instance.perform()
[body_behavior-32]   File "/colcon_ws/install/dynamic_stack_decider/lib/python3.10/site-packages/dynamic_stack_decider/sequence_element.py", line 28, in perform
[body_behavior-32]     self.current_action.perform()
[body_behavior-32]   File "/colcon_ws/install/bitbots_body_behavior/share/bitbots_body_behavior/actions/go_to_block_position.py", line 45, in perform
[body_behavior-32]     pose_msg.pose.orientation.w = 1
[body_behavior-32]   File "/opt/ros/rolling/local/lib/python3.10/dist-packages/geometry_msgs/msg/_quaternion.py", line 214, in w
[body_behavior-32]     assert \
[body_behavior-32] AssertionError: The 'w' field must be of type 'float'
```

## Steps to Reproduce
<!--- An unambiguous set of steps to reproduce this bug. -->
<!--- Code-snippets, screenshots ot other details are welcome if needed. -->
1.
2.
3.
...

## Context (Environment)
HLVS Setup, see here: https://data.bit-bots.de/HLVS/2023/GAME_K-GD1-1_2023-03-17-07__itandroids_bitbots_99d633fc05/private_logs/ID_1/docker.log

## Possible Solution
<!--- NOT OBLIGATORY -->
<!--- If you already have an idea, you can suggest a fix/reason for the bug -->

--- a/bitbots_body_behavior/bitbots_body_behavior/actions/go_to_block_position.py
+++ b/bitbots_body_behavior/bitbots_body_behavior/actions/go_to_block_position.py
@@ -40,9 +40,9 @@ class GoToBlockPosition(AbstractActionElement):
         pose_msg.header.stamp = self.blackboard.node.get_clock().now().to_msg()
         pose_msg.header.frame_id = self.blackboard.map_frame
 
-        pose_msg.pose.position.x = -(self.blackboard.world_model.field_length / 2) + self.block_position_goal_offset
-        pose_msg.pose.position.y = self._stay_in_front_of_goal(goalie_y)
-        pose_msg.pose.orientation.w = 1
+        pose_msg.pose.position.x = float(-(self.blackboard.world_model.field_length / 2) + self.block_position_goal_offset)
+        pose_msg.pose.position.y = float(self._stay_in_front_of_goal(goalie_y))
+        pose_msg.pose.orientation.w = 1.0
 
         self.blackboard.pathfinding.publish(pose_msg)
 

